nvflare/private/fed/app/fl_conf.py

```diff
@@ -39,13 +40,15 @@
 class FLServerStarterConfiger(JsonConfigurator):
     """FL Server startup configer."""
 
-    def __init__(self, workspace: Workspace, kv_list=None):
+    def __init__(self, workspace: Workspace, args, kv_list=None):
```

Comment:
in the later re-factor, we might want to explicitly specify what needs to be passed in.
instead of a random argument call `args` where people can put anything inside.

Reply:
I agree we can re-factor this to make it more meaningful. That may require more code analysis and extraction.